### ISSUE TITLE
Fix docstring errors in registration.py, symbolic_helper.py, __init__.py, _property_propagation.py, io_adapter.py, symbolic_opset17.py, symbolic_opset14.py, onnx_proto_utils.py, symbolic_opset9.py, _beartype.py, utils.py, symbolic_opset11.py, _type_utils.py, symbolic_opset13.py, symbolic_opset8.py, _onnx_supported_ops.py, _deprecation.py, verification.py, symbolic_opset10.py, symbolic_opset7.py

### ISSUE THREAD

**Issue by svekars** [svekars/test-stats#268]
Please fix the following issues. To test locally, run: 
 ```pydocstyle path-to-file --count``` 
 Run this command before and after fixing the errors. This command prints out the number of errors in the end of the output. Paste the output to your PR description with the number before and after. 
- **File**: `torch/jit/_passes/_property_propagation.py`, **Entity**: `apply_input_props_using_example`, **Line**: 16, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/jit/_passes/_property_propagation.py`, **Entity**: `apply_input_props_using_example`, **Line**: 16, **Description**: First line should end with a period (not 's')
- **File**: `torch/jit/_passes/_property_propagation.py`, **Entity**: `apply_input_props_using_example`, **Line**: 16, **Description**: First line should be in imperative mood (perhaps 'Apply', not 'Applies')
- **File**: `torch/onnx/_onnx_supported_ops.py`, **Entity**: `all_forward_schemas`, **Line**: 70, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/_onnx_supported_ops.py`, **Entity**: `all_symbolics_schemas`, **Line**: 76, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/symbolic_opset7.py`, **Entity**: ``, **Line**: 1, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/symbolic_opset7.py`, **Entity**: ``, **Line**: 1, **Description**: First line should end with a period (not ']')
- **File**: `torch/onnx/__init__.py`, **Entity**: `enable_log`, **Line**: 124, **Description**: First line should be in imperative mood (perhaps 'Enable', not 'Enables')
- **File**: `torch/onnx/_deprecation.py`, **Entity**: `deprecated`, **Line**: 9, **Description**: First line should be in imperative mood (perhaps 'Mark', not 'Marks')
- **File**: `torch/onnx/symbolic_opset14.py`, **Entity**: `_causal_attention_mask`, **Line**: 247, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/onnx/symbolic_opset17.py`, **Entity**: `_compute_edge_sizes`, **Line**: 61, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/symbolic_opset17.py`, **Entity**: `_compute_edge_sizes`, **Line**: 61, **Description**: Multi-line docstring closing quotes should be on a separate line
- **File**: `torch/onnx/symbolic_opset17.py`, **Entity**: `_compute_edge_sizes`, **Line**: 61, **Description**: First line should end with a period (not ')')
- **File**: `torch/onnx/symbolic_opset17.py`, **Entity**: `_compute_edge_sizes`, **Line**: 61, **Description**: First line should be in imperative mood; try rephrasing (found 'Helper')
- **File**: `torch/onnx/symbolic_opset17.py`, **Entity**: `stft`, **Line**: 82, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/symbolic_opset8.py`, **Entity**: ``, **Line**: 1, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/symbolic_opset8.py`, **Entity**: ``, **Line**: 1, **Description**: First line should end with a period (not ']')
- **File**: `torch/onnx/symbolic_opset8.py`, **Entity**: `_apply_params`, **Line**: 68, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/verification.py`, **Entity**: `_try_clone_model`, **Line**: 396, **Description**: First line should be in imperative mood (perhaps 'Use', not 'Used')
- **File**: `torch/onnx/verification.py`, **Entity**: `export_repro`, **Line**: 1352, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/onnx/utils.py`, **Entity**: `is_in_onnx_export`, **Line**: 71, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/utils.py`, **Entity**: `select_model_mode_for_export`, **Line**: 83, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/utils.py`, **Entity**: `select_model_mode_for_export`, **Line**: 83, **Description**: First line should end with a period (not '`')
- **File**: `torch/onnx/utils.py`, **Entity**: `select_model_mode_for_export`, **Line**: 83, **Description**: First line should be in imperative mood; try rephrasing (found 'A')
- **File**: `torch/onnx/utils.py`, **Entity**: `export`, **Line**: 209, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/onnx/utils.py`, **Entity**: `export`, **Line**: 209, **Description**: First line should be in imperative mood (perhaps 'Export', not 'Exports')
- **File**: `torch/onnx/utils.py`, **Entity**: `_resolve_args_by_export_type`, **Line**: 738, **Description**: First line should be in imperative mood (perhaps 'Resolve', not 'Resolves')
- **File**: `torch/onnx/utils.py`, **Entity**: `_decide_keep_init_as_input`, **Line**: 759, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/onnx/utils.py`, **Entity**: `_pre_trace_quant_model`, **Line**: 1070, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/utils.py`, **Entity**: `_pre_trace_quant_model`, **Line**: 1070, **Description**: First line should end with a period (not 'n')
- **File**: `torch/onnx/utils.py`, **Entity**: `_pre_trace_quant_model`, **Line**: 1070, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/utils.py`, **Entity**: `_model_to_graph`, **Line**: 1108, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/onnx/utils.py`, **Entity**: `export_to_pretty_string`, **Line**: 1246, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/utils.py`, **Entity**: `export_to_pretty_string`, **Line**: 1246, **Description**: First line should end with a period (not 'X')
- **File**: `torch/onnx/utils.py`, **Entity**: `unconvertible_ops`, **Line**: 1311, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/onnx/utils.py`, **Entity**: `unconvertible_ops`, **Line**: 1311, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/utils.py`, **Entity**: `_run_symbolic_method`, **Line**: 1714, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/utils.py`, **Entity**: `_run_symbolic_method`, **Line**: 1714, **Description**: First line should end with a period (not 'd')
- **File**: `torch/onnx/utils.py`, **Entity**: `_run_symbolic_method`, **Line**: 1714, **Description**: First line should be in imperative mood; try rephrasing (found 'This')
- **File**: `torch/onnx/utils.py`, **Entity**: `_need_symbolic_context`, **Line**: 1785, **Description**: First line should be in imperative mood (perhaps 'Check', not 'Checks')
- **File**: `torch/onnx/utils.py`, **Entity**: `_symbolic_context_handler`, **Line**: 1801, **Description**: First line should be in imperative mood (perhaps 'Decorate', not 'Decorator')
- **File**: `torch/onnx/utils.py`, **Entity**: `_run_symbolic_function`, **Line**: 1842, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/onnx/utils.py`, **Entity**: `_run_symbolic_function`, **Line**: 1842, **Description**: First line should be in imperative mood (perhaps 'Run', not 'Runs')
- **File**: `torch/onnx/utils.py`, **Entity**: `register_custom_op_symbolic`, **Line**: 1975, **Description**: First line should be in imperative mood (perhaps 'Register', not 'Registers')
- **File**: `torch/onnx/utils.py`, **Entity**: `_validate_dynamic_axes`, **Line**: 2027, **Description**: First line should be in imperative mood (perhaps 'Ensure', not 'Ensures')
- **File**: `torch/onnx/symbolic_helper.py`, **Entity**: `_node_get`, **Line**: 137, **Description**: First line should be in imperative mood (perhaps 'Get', not 'Gets')
- **File**: `torch/onnx/symbolic_helper.py`, **Entity**: `_unpack_quantized_tensor`, **Line**: 207, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/symbolic_helper.py`, **Entity**: `parse_args`, **Line**: 236, **Description**: First line should be in imperative mood; try rephrasing (found 'A')
- **File**: `torch/onnx/symbolic_helper.py`, **Entity**: `quantized_args`, **Line**: 319, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/symbolic_helper.py`, **Entity**: `quantized_args`, **Line**: 319, **Description**: First line should be in imperative mood; try rephrasing (found 'A')
- **File**: `torch/onnx/symbolic_helper.py`, **Entity**: `_if_scalar_type_as`, **Line**: 455, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/symbolic_helper.py`, **Entity**: `_is_scalar_list`, **Line**: 518, **Description**: First line should be in imperative mood (perhaps 'Check', not 'Checks')
- **File**: `torch/onnx/symbolic_helper.py`, **Entity**: `_generate_wrapped_number`, **Line**: 753, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/onnx/symbolic_helper.py`, **Entity**: `dequantize_helper`, **Line**: 1573, **Description**: First line should be in imperative mood (perhaps 'Append', not 'Appends')
- **File**: `torch/onnx/symbolic_helper.py`, **Entity**: `quantize_helper`, **Line**: 1624, **Description**: First line should be in imperative mood (perhaps 'Append', not 'Appends')
- **File**: `torch/onnx/symbolic_helper.py`, **Entity**: `requantize_bias_helper`, **Line**: 1682, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/symbolic_opset13.py`, **Entity**: `_apply_params`, **Line**: 25, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/symbolic_opset11.py`, **Entity**: `_apply_params`, **Line**: 90, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `_apply_params`, **Line**: 295, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `_export`, **Line**: 304, **Description**: First line should be in imperative mood (perhaps 'Export', not 'Exports')
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `unused`, **Line**: 316, **Description**: First line should be in imperative mood (perhaps 'Represent', not 'Represents')
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `true_divide`, **Line**: 493, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `true_divide`, **Line**: 493, **Description**: First line should end with a period (not 's')
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `_op_with_optional_float_cast`, **Line**: 1358, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `_op_with_optional_float_cast`, **Line**: 1358, **Description**: First line should be in imperative mood; try rephrasing (found 'Some')
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `_prepare_onnx_paddings`, **Line**: 1912, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `nonzero`, **Line**: 5311, **Description**: First line should end with a period (not '`')
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `nonzero`, **Line**: 5311, **Description**: First line should be in imperative mood (perhaps 'Emit', not 'Emitted')
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `nonzero`, **Line**: 5311, **Description**: First line should not be the function's "signature"
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `dim`, **Line**: 6241, **Description**: First line should end with a period (not 'X')
- **File**: `torch/onnx/symbolic_opset9.py`, **Entity**: `prim_tolist`, **Line**: 6833, **Description**: First word of the first line should be properly capitalized ('Tolist', not 'tolist')
- **File**: `torch/onnx/_type_utils.py`, **Entity**: `from_value`, **Line**: 153, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/onnx/symbolic_opset10.py`, **Entity**: `_apply_params`, **Line**: 71, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/_internal/_beartype.py`, **Entity**: `beartype`, **Line**: 75, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/onnx/_internal/onnx_proto_utils.py`, **Entity**: `_export_file`, **Line**: 170, **Description**: First line should end with a period (not 'p')
- **File**: `torch/onnx/_internal/onnx_proto_utils.py`, **Entity**: `_add_onnxscript_fn`, **Line**: 218, **Description**: First line should end with a period (not 'o')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `_dispatch_opset_version`, **Line**: 25, **Description**: First line should be in imperative mood (perhaps 'Find', not 'Finds')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `in_base`, **Line**: 82, **Description**: First line should be in imperative mood (perhaps 'Check', not 'Checks')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `override`, **Line**: 86, **Description**: First line should be in imperative mood (perhaps 'Override', not 'Overrides')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `overridden`, **Line**: 98, **Description**: First line should be in imperative mood (perhaps 'Check', not 'Checks')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `add`, **Line**: 160, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `add_custom`, **Line**: 176, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `remove_custom`, **Line**: 185, **Description**: First line should be in imperative mood (perhaps 'Remove', not 'Removes')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `get_min_supported`, **Line**: 198, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `register`, **Line**: 216, **Description**: First line should be in imperative mood (perhaps 'Register', not 'Registers')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `get_function_group`, **Line**: 252, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `is_registered_op`, **Line**: 256, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `all_functions`, **Line**: 263, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `onnx_symbolic`, **Line**: 274, **Description**: First line should be in imperative mood (perhaps 'Register', not 'Registers')
- **File**: `torch/onnx/_internal/registration.py`, **Entity**: `custom_onnx_symbolic`, **Line**: 322, **Description**: First line should be in imperative mood (perhaps 'Register', not 'Registers')
- **File**: `torch/onnx/_internal/io_adapter.py`, **Entity**: `append_step`, **Line**: 53, **Description**: First line should be in imperative mood (perhaps 'Append', not 'Appends')
- **File**: `torch/onnx/_internal/io_adapter.py`, **Entity**: `apply`, **Line**: 64, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/onnx/_internal/io_adapter.py`, **Entity**: `append_step`, **Line**: 106, **Description**: First line should be in imperative mood (perhaps 'Append', not 'Appends')
- **File**: `torch/onnx/_internal/io_adapter.py`, **Entity**: `apply`, **Line**: 117, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
